modules/organizations-and-accounts/exercises.md

```diff
@@ -40,4 +40,6 @@ with the following name: ${kerberosID}-challenge-3.
 Move the AWS account from #1 into the OU you created from #3.
 
 ## Challenge 5
-Move the new OU from #3 into the following OU: ou-rs3h-1pslrete
+Create a new OU with ou-rs3h-1pslrete as the parent
+Move the account you identified from #1 into the new OU 
+Delete the OU you created in Challenge 3
```

Comment:
This was just for fixing the wrong instructions. Instead, lets add something about tagging the OU with some tags in challenge 5